functionary/core/models/workflow_step.py

```diff
@@ -102,7 +102,6 @@ def execute(self, workflow_task: "Task") -> "Task":
 
         with transaction.atomic():
             run_context = self._get_run_context(workflow_task)
-            workflow_task.in_progress()
```

Comment:
Realized when removing the usage of these helpers that this one was not really needed or appropriate, so it's just been removed rather than replaced with an explicit setting of status.